site/plans.html.jet

```diff
@@ -59,9 +66,16 @@
                   {{ if p.Page }}
                     <a class="plan-page-link btn btn-light text-dark btn-block" href="{{ routeToSlug(p.Page.Slug) }}">{{i18n("plan_read_more")}}</a>
                   {{ end }}
-                  <s72-pricing-buttons class="btn-block" data-slug="{{p.Slug}}" data-title="{{p.Name}}" />
+                  {{ if p.PlanType == "free" }}
```

Comment:
Maybe discuss while I was away, but are we allowing users to sign up even if the plan has already expired? Im guessing these buttons display after expiry date?

Reply:
We could wrap this in a <s72-before> tag so the CTA won't appear after the expiry date.

Reply:
@ashesnz Done